packages/lit-dev-content/src/icons/labs-icon.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {html} from 'lit';
+
+export const labsIcon = html`<svg
+  xmlns="http://www.w3.org/2000/svg"
+  enable-background="new 0 0 24 24"
```

Comment:
```suggestion
export const labsIcon = html`<svg
  enable-background="new 0 0 24 24"
```

xmlns not needed for inline svgs